### DISCUSSION TITLE
Error 404 on my website (host GitHub)

### DISCUSSION THREAD

**Discussion by Fernando2008** [community/community#51253] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

A few hours ago, I wanted to publish my website on internet. So I created a new repository, I uploaded my two files (First is Html with Js inside and the other is Css) on it, I went on GitHub Pages , I chose "Deploy from a Branch" on source and I also chose "main" and "/root" on "Branch" department. Then I saved it and after few minutes GitHub deployed my website on internet but when I click on the link GitHub gave to me, my website opens but I get this written "
404

File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.

[GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)". Can someone help me pls?

**Comment by matej-kaska** [comment]
Your HTML file in your repository should be named "index.html". Then it should redirect you to your HTML site with GitHub Pages.

**Reply by Fernando2008** [reply]
I'm going to do it this afternoon, anyway thanks.

**Reply by matej-kaska** [reply]
No problem, if it helps, feel free to accept my answer. :)
